tests/AiDotNet.Tensors.Benchmarks/BaselineRunners/py/run_direct_ptx_vision_competitors.py

```
@@ -0,0 +1,389 @@
+
+def available(name: str) -> Callable | None:
+    return getattr(ops, name, None)
+
+
+def maximum_error(actual: object, reference: object) -> float:
+    actual_values = actual if isinstance(actual, (tuple, list)) else (actual,)
+    reference_values = reference if isinstance(reference, (tuple, list)) else (reference,)
+    if len(actual_values) != len(reference_values):
+        return math.inf
+    error = 0.0
+    for actual_value, reference_value in zip(actual_values, reference_values):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)